internal/backend/gs/gs.go

```diff
@@ -282,8 +293,13 @@ func (be *Backend) Load(ctx context.Context, h restic.Handle, length int, offset
 }
 
 func (be *Backend) openReader(ctx context.Context, h restic.Handle, length int, offset int64) (io.ReadCloser, error) {
+	var err error
+	var rangeReader *storage.Reader
```

Comment:
Don't predefine variables, but rather define them near the place they are used for the first time.

Reply:
I tried to define variables closer to where they are used.